docs/gum-tool/setup/README.md

```diff
@@ -161,67 +155,70 @@ sudo apt-get install winetricks
 
 These following commands will go through the steps setting up your macOS or Linux environment to run the GUM tool using WINE.
 
-1. Open a new terminal
-2. Install .NET Framework 4.8 using `winetricks` with the following command:
+1. Open a new terminal and set our wine prefix for Gum with the following command:
 
 ```sh
-winetricks dotnet48
+GUM_WINE_PREFIX_PATH=$HOME/.wine_gum_prefix/
 ```
 
-This command initiates the installation of .NET Framework 4.8. A total of two installer dialogs appear one after another. Follow the steps for both to complete the installation. **This process may take a several minutes, so please be patient**.
-
-1. Download the XNA 4.0 Redistributable MSI file from Microsoft. You can download it using the following command:
+2. Install Windows All Fonts using `winetricks` with the following command:
 
 ```sh
-curl -O https://download.microsoft.com/download/A/C/2/AC2C903B-E6E8-42C2-9FD7-BEBAC362A930/xnafx40_redist.msi
+WINEPREFIX=$GUM_WINE_PREFIX_PATH winetricks allfonts
 ```
 
-4. After downloading the MSI file, install it using the following command:
+3. Install .NET Framework 4.8 using `winetricks` with the following command:
 
 ```sh
-wine msiexec /i xnafx40_redist.msi
+WINEPREFIX=$GUM_WINE_PREFIX_PATH winetricks dotnet48
 ```
 
-Follow the installation prompts. At the end of the installation, it may display an error related to launching DirectX. **This is normal; just click "Close" on the error dialog.**
+This command initiates the installation of .NET Framework 4.8. A total of two installer dialogs appear one after another. Follow the steps for both to complete the installation. **This process may take a several minutes, so please be patient**.
 
-5. Download the Gum Tool ZIP file from the FlatRedBall website and save it to your preferred location. You can download it using the following command:
+4. Download the XNA 4.0 Redistributable MSI file from Microsoft. You can download it using the following command:
```

Comment:
@vchelaru I thought xna is no longer required?

Reply:
Correct, XNA is no longer required.

Reply:
Ya, removing would be good